run-tests.sh

```diff
@@ -1,14 +1,69 @@
-#!/bin/sh
+#!/bin/bash
 #
 # This file is part of REANA.
-# Copyright (C) 2018 CERN.
+# Copyright (C) 2018, 2019, 2020 CERN.
 #
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-pydocstyle reana_db && \
-black --check . && \
-check-manifest --ignore ".travis-*" && \
-sphinx-build -qnNW docs docs/_build/html && \
-REANA_SQLALCHEMY_DATABASE_URI=sqlite:// python setup.py test && \
+# Quit on errors
+set -o errexit
+
+# Quit on unbound symbols
+set -o nounset
+
+export REANA_SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://postgres:mysecretpassword@localhost/postgres
+
+# Verify that db container is running before continuing
+_check_ready() {
+    RETRIES=40
+    while ! $2
+    do
+        echo "==> [INFO] Waiting for $1, $((RETRIES--)) remaining attempts..."
+        sleep 2
+        if [ $RETRIES -eq 0 ]
+        then
+            echo "==> [ERROR] Couldn't reach $1"
+            exit 1
+        fi
+    done
+}
+
+_db_check() {
+    docker exec --user postgres postgres__reana-db bash -c "pg_isready" &>/dev/null;
+}
+
+clean_old_db_container() {
+    OLD="$(docker ps --all --quiet --filter=name=postgres__reana-db)"
+    if [ -n "$OLD" ]; then
+        echo '==> [INFO] Cleaning old DB container...'
+        docker stop postgres__reana-db
```

Comment:
BTW is it enough to `stop` the old container? Shouldn't we also `rm` it, just to be sure that it is created anew and does not carry over any past state?  The current thing is obviously OK for Travis CI runs, where everything is run on a blank slate VM, but if one uses `run-tests.sh` script locally, then one ends with a lot of stopped (and never removed) local containers that could carry over some unwanted state. Perhaps we should do `rm` instead of `stop` here and also at the end of tests, to end up with a blank slate again?

(This applies also to the other SQLite->PostgreSQL PRs.)

Reply:
`docker run` has already `--rm` so it's removed automatically when stopped.